object/address/address_test.go

```diff
@@ -1,14 +1,31 @@
-package object
+package address
 
 import (
+	"crypto/rand"
+	"crypto/sha256"
 	"strings"
 	"testing"
 
 	"github.com/nspcc-dev/neofs-api-go/v2/refs"
 	cidtest "github.com/nspcc-dev/neofs-sdk-go/container/id/test"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
 	"github.com/stretchr/testify/require"
 )
 
+func randID(t *testing.T) *oid.ID {
```

Comment:
We can't import `oidtest github.com/nspcc-dev/neofs-sdk-go/object/id/test"` and reuse `ID()` because of cycle dependency?

Reply:
in fact no, just forget to reuse it after creating new `test` pkgs in the `id` and `address`

fixed